verification/curator-service/ui/src/components/BulkCaseForm.tsx

```diff
@@ -523,30 +523,27 @@ class BulkCaseForm extends React.Component<
             );
             return;
         }
-        const createdIds = upsertResponse.createdCaseIds;
-        const updatedIds = upsertResponse.updatedCaseIds;
         await this.finalizeUpload(caseReference.sourceId, uploadId, 'SUCCESS', {
-            numCreated: createdIds.length,
-            numUpdated: updatedIds.length,
+            numCreated: upsertResponse.numCreated,
+            numUpdated: upsertResponse.numUpdated,
         });
         const createdMessage =
-            createdIds.length === 0
+            upsertResponse.numCreated === 0
                 ? ''
-                : createdIds.length === 1
+                : upsertResponse.numCreated === 1
                 ? '1 new case added. '
-                : `${createdIds.length} new cases added. `;
+                : `${upsertResponse.numCreated} new cases added. `;
         const updatedMessage =
-            updatedIds.length === 0
+            upsertResponse.numUpdated === 0
                 ? ''
-                : updatedIds.length === 1
+                : upsertResponse.numUpdated === 1
                 ? '1 case updated. '
-                : `${updatedIds.length} cases updated. `;
+                : `${upsertResponse.numUpdated} cases updated. `;
         this.props.history.push({
             pathname: '/cases',
             state: {
                 bulkMessage: `${filename} uploaded. ${createdMessage} ${updatedMessage}`,
-                newCaseIds: createdIds,
-                editedCaseIds: updatedIds,
+                searchQuery: `uploadid:${uploadId}`,
```

Comment:
Will this search result include cases that were matched but not updated?

Reply:
It will -- the behavior is ~unchanged as compared to before this PR. We have the option, after this, to remove adding the upload ID to unchanged cases: which will help with the verification process.